src/Remark.jl

```diff
@@ -22,8 +22,12 @@ const depfiles = joinpath.(_pkg_assets, depnames)
 
 const styles_css = joinpath(_pkg_assets, "styles.css")
 
-function slideshow(inputfile, outputdir = dirname(inputfile);
-    title = "Title", documenter = true, css = styles_css, options = Dict())
+function slideshow(inputfile, outputdir=dirname(inputfile);
+    title="Title", documenter=true, css=styles_css, options=Dict())
+
+    inputfile = realpath(abspath(inputfile))
+    css = realpath(abspath(css))
+    assets_dir = joinpath(dirname(inputfile), "assets")
```

Comment:
Maybe it would be ok to just take `inputdir` and the inputfule has to be calles `index.md` / `index.jl`, and the css has to be called `style.css` and the assets have to be in `assets`

Reply:
So, with your trick of using a tempdir, we don't need the `src` directory for Documenter (it all happens in the temp folder), so a possible structure would be:

```
presentation/
├── src/
│   └── index.md (or .jl)
│   └── style.css
│   └── assets/
├── build/
│   └── index.html (generated by the package)
│   └── style.css
│   └── assets/
```

Potentially the user could completely mirror documenter and add a `make.jl` file with a reloading script.

So this would essentially be a breaking change where the `inputfile`, `outputdir`, and `css` arguments are replaced by a unique `input_dir` argument. OTOH, I don't think there is a strong reason to make this customizable. This way, Remark can also offer a function that does the reloading (it's enough to watch the folder `src`).

Reply:
I guess when doing this one maybe should think if there are other things wanted that could/should be done.
E.g. should it actually be a directory of stylesheets, all of which would be included?

For that matter, should one be allowed multiple markdown files that would be concatenated in lexigraphical order during processing?

Reply:
So, I've never needed multiple markdown files, and I'm afraid it could get a bit confusing.

OTOH, I often need two stylesheets, one is basically a customized Remark theme, and the other is for layouting (a bunch of flex containers essentially). Maybe the simplest rule is that if there is a file called `style.css`, use that one, if not, see if there is a folder `style/` and use all `css` in there, otherwise, use default css.